docs/source/support/release-notes.rst

```diff
@@ -9,6 +9,37 @@ This page contains release notes for the SDK.
 HEAD — ongoing
 --------------
 
+Ledger API
+~~~~~~~~~~
+- **BREAKING** Removed the unused field :ref:`com.digitalasset.ledger.api.v1.ExercisedEvent` from :ref:`com.digitalasset.ledger.api.v1.Event`.
+  ``Event`` is only used in :ref:`com.digitalasset.ledger.api.v1.Transaction`, which in turn by definition never contains exercised events (only created and archived events): `#960 <https://github.com/digital-asset/daml/issues/960>`_
+
+  This change is *backwards compatible on the transport level*, meaning:
+
+  - new versions of ledger language bindings will work with previous versions of the sandbox, as the field was never populated,
```

Comment:
```suggestion
  - new versions of ledger language bindings will work with previous versions of the Sandbox, because the field was never populated
```